tools/mech_converter.py

```diff
@@ -191,7 +191,7 @@ def convert(input_file, output_dir, mc_dir, mcm_dir):
 
     # check RO2s are in RO2 list
     # Read in RO2 list, and strip off newlines
-    with open(os.path.join(mcm_dir, 'RO2listv3.3.1')) as RO2List_file:
```

Comment:
should it be : `with open(os.path.join(output_dir, 'gen/mechanism-rate-coefficients.f90')`

Reply:
No, because in `setup_atchem2.py` line 31 sets `output_dir` to `src/gen` already - that is abstracted away from this script.